### PR TITLE
voting for sources

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

**Comment by xithan** [comment]
commited code for contributions by accident

File: db/migrate_cards/20141023120538_direct_contributions.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# -*- encoding : utf-8 -*-
+
+class DirectContributions < ActiveRecord::Migration
+  include Wagn::MigrationHelper
+  def up
+    contentedly do
+      Card.create! :name=>"*direct contribution count", :codename=>:direct_contribution_count, :type_code=>:number, :content=>"0"
+    end
+  end
+end
```
Comment (ethn) [inline db/migrate_cards/20141023120538_direct_contributions.rb:10]: cool.  are migrations now set up to run both deck and gem migrations?

**Comment by ethn** [comment]
lots of really good stuff here.

File: db/migrate_cards/20141023120538_direct_contributions.rb (hunk 1 above)
Comment (xithan) [inline db/migrate_cards/20141023120538_direct_contributions.rb:10]: yes  `rake wagn:migrate` runs both

Comment (ethn) [inline db/migrate_cards/20141023120538_direct_contributions.rb:10]: I looked more closely at the code that runs this in the other PR.  very cool.  did you test to see if anything breaks in a normal deck?  (I didn't see any red flags; just curious)

**Pull request closed without merging**